js/ui/tooltip-base.js

```diff
@@ -0,0 +1,171 @@
+import $ from "jquery";
+import Backbone from "backbone";
+Backbone.$ = $;
+
+import "../../css/ui/tooltips.css";
+
```

Comment:
The file split from tooltip-engine.

Reply:
Cool.  I'm going to assume that there are no changes `TooltipBase`.